default-fields/0000-default-fields.md

```diff
@@ -6,31 +6,207 @@
 # Summary
 [summary]: #summary
 
-One para explanation of the feature.
+Allow structure definitions to supply default values for individual fields, and then allow those fields to be omitted from struct initialisation:
+
+```rust
+struct Foo {
+    a: &'static str,
+    b: bool = true,
+    c: i32,
+}
+
+let foo = Foo {
+    a: "Hello",
+    c: 42,
+}
+```
 
 # Motivation
 [motivation]: #motivation
 
-Why are we doing this? What use cases does it support? What is the expected outcome?
+Field default address two issues with structure initialisation in Rust: privacy and boilerplate. This is achieved by letting callers omit fields from initialisation when a default is specified for that field. This syntax also allows structures to be updated in a non-breaking fashion, by providing defaults for new fields.
```

Comment:
"updated" is a bit ambiguous here because of the term "functional record update", maybe just say "allows fields to be added to structs in a backwards compatible way"